test/builder/QuarkBuilderCometWithdraw.t.sol

```diff
@@ -253,23 +281,32 @@ contract QuarkBuilderCometWithdrawTest is Test, QuarkBuilderTest {
         Accounts.ChainAccounts[] memory chainAccountsList = new Accounts.ChainAccounts[](2);
         chainAccountsList[0] = Accounts.ChainAccounts({
             chainId: 1,
-            quarkStates: quarkStates_(address(0xa11ce), 12),
+            quarkSecrets: quarkSecrets_(address(0xa11ce), ALICE_DEFAULT_SECRET),
             assetPositionsList: assetPositionsList_(1, address(0xa11ce), 3e6), // 3 USDC on mainnet
             cometPositions: emptyCometPositions_(),
             morphoPositions: emptyMorphoPositions_(),
             morphoVaultPositions: emptyMorphoVaultPositions_()
         });
         chainAccountsList[1] = Accounts.ChainAccounts({
             chainId: 8453,
-            quarkStates: quarkStates_(address(0xb0b), 2),
+            quarkSecrets: quarkSecrets_(address(0xb0b), BOB_DEFAULT_SECRET),
             assetPositionsList: assetPositionsList_(8453, address(0xb0b), 0), // 0 USDC on base
             cometPositions: emptyCometPositions_(),
             morphoPositions: emptyMorphoPositions_(),
             morphoVaultPositions: emptyMorphoVaultPositions_()
         });
 
         QuarkBuilder.BuilderResult memory result = builder.cometWithdraw(
-            cometWithdraw_(8453, cometUsdc_(8453), "LINK", 5e18), chainAccountsList, paymentUsdc_(maxCosts)
+            // We need to set Bob as the withdrawer because only he has an account on chain 8453
+            cometWithdraw_({
+                chainId: 8453,
+                comet: cometUsdc_(8453),
+                assetSymbol: "LINK",
+                amount: 5e18,
+                withdrawer: address(0xb0b)
+            }),
```

Comment:
Example of the nonce bug in action. Previously, we were withdrawing using `0xa11ce` as the withdrawer on chain `8453`, even though a nonce for `0xa11ce` wasn't provided for chain 8453. Now, we have to set the withdrawer as `0xb0b` since only he has a nonce on chain 8453.